<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogWithdrawlToTake</class>
 <widget class="QDialog" name="DialogWithdrawlToTake">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>531</width>
     <height>43</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QToolButton" name="YearButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>年份</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="yearBox">
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="minimum">
       <number>1900</number>
      </property>
      <property name="maximum">
       <number>2021</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="MonthButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>月份</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="monthBox">
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>12</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="DayButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>日期</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="dayBox">
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>31</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="CheckButton">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>681</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>709</width>
     <height>43</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>账号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="sortName">
      <property name="text">
       <string>姓名</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>密码</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>地址</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="sortType">
      <property name="text">
       <string>储种</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="sortCapital">
      <property name="text">
       <string>本金</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="sortDate">
      <property name="text">
       <string>到息日期</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="sortLost">
      <property name="text">
       <string>是否挂失</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>挂失日期</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>营业员姓名</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>营业员工号</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
